.idea/misc.xml

```diff
@@ -1,8 +1,5 @@
 <?xml version="1.0" encoding="UTF-8"?>
```

Comment:
@chetan-fueled let's add these two lines to our `.gitignore` and remove everything inside the `idea` folder except for `codeStyleSettings.xml`

```
.idea/*
!.idea/codeStyleSettings.xml
```